projects/GEDKeeper2/GKUI/BaseWin.cs

```diff
@@ -1376,55 +1455,119 @@ public bool RecordDelete(GEDCOMRecord record, bool confirm)
             {
                 //string xref = record.XRef;
                 string msg = "";
-
                 switch (record.RecordType)
                 {
                     case GEDCOMRecordType.rtIndividual:
-                        msg = string.Format(LangMan.LS(LSID.LSID_PersonDeleteQuery), ((GEDCOMIndividualRecord)record).GetNameString(true, false));
+                    {
+                        msg = string.Format(LangMan.LS(
+                            LSID.LSID_PersonDeleteQuery),
+                            ((GEDCOMIndividualRecord)record).GetNameString(
+                                true, false));

```

Comment:
As you wish. But you definitely exceed 80-characters per line limit.

I revert the code format to your initial style -- one LONG expression per one line. I hate that, just like I hate this:

```csharp
msg = string.Format(LangMan.LS(LSID.LSID_PersonDeleteQuery),
                            ((GEDCOMIndividualRecord)record).GetNameString(true, false));
```

Hereinafter 😆